README.md

```diff
@@ -22,49 +34,149 @@ You can do it manually following the instructions from [rust-lang.org](https://w
 
-```
-mkdir build && cd build
-cmake -DCLANG_TIDY=ON ..
-make -j
-```
 
-It is suggested to build project with clang-tidy checks, however if you wish to omit clang-tidy step, you can use `cmake ..` instead.
+## Configuration Lotus-Fuhon
+### Lotus-node and Fuhon-miner
+1. To start up a lotus node at first you need to get [lotus executable](https://github.com/filecoin-project/lotus/releases/ "lotus releases")
+2. You can run your lotus node with the next command: `$lotus --repo <node working directory> daemon --genesis genesis.car --import-key PRESEAL/pre-seal-MINER.key --profile bootstrapper`, more about running lotus software can be found in [filecoin-docs](https://docs.filecoin.io/get-started/lotus/installation "lotus filecoin documentation").
 
-Tests can be run with: 
-```
-cd build
-ctest
-```
+### Fuhon-node and Lotus-miner
+1. Establishing of a Fuhon node is described in **Fuhon miner establishing**
+2. Lotus miner can be established by the following way:
+    - to begin with you need you own bls wallet(described in **Working with Lotus-CLI**)
+    - run your miner with command: `lotus-miner --repo <your node running directory> --miner-repo <minner future working directory> init --nosync --owner <your bls wallet public address> --sector-size <Size of sectors that will be sealed>`
+    - to know more about lotus software establishing, read [filecoin-docs](https://docs.filecoin.io/get-started/lotus/installation/#minimal-requirements "lotus establishing guide")
 
-### CodeStyle
+## Docker configuration
+    Will be soon!
 
-We follow [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines).
+## Your new Filecoin network start
 
-Please use clang-format 11.0.0 with provided [.clang-format](.clang-format) file to autoformat the code.  
+To run a new filecoin network, you should run at least two nodes, that will have comunication with each other and a miner (miners are representation of the network power, so more miners connected and sealed, more power your network will have).
 
-## Maintenance
+1. To run the first node you will need to generate a new genesis file which will be an initial instruction for all new conected nodes.
+   This can be done with lotus-seed tools:
+    - Clone  [lotus repository](https://github.com/filecoin-project/lotus "lotus project")
 
-Maintainers: @kamilsa, @zuiris, @harrm, @masterjedy, @Mingela, @Warchant
+    - build lotus-seed with `$make lotus-seed`
 
-Tickets: Can be opened in Github Issues.
+    - Generate genesis and preseal. Ensure lotus-seed was build in previous step and run script node `$genesis.js DIR 1` to generate genesis. `DIR` - directory where genesis and presealed sectors will be generated, 1 - number of sectors for miner. In case your lotus executables lay in custom directory or sript can not find them, you can link lotus target with `LOTUS=/<path to your lotus executable>/lotus node genesis.js <directory where results wil be> <sealing sectors upper bound>`
```

Comment:
there is no genesis.js. can write bash commands here.

is this section required? will it help users? is it sufficient?

Reply:
this section will help users to establish their network, without genesis it can be done with genesis of other networks, which make new network predictable at the initial state